cmd/messaging-tool/receive.go

```diff
@@ -106,10 +103,13 @@ func runReceive(cmd *cobra.Command, args []string) {
 		destinationName,
 	)
 
-	// Wait for Ctrl+C.
-	waitCtrlC := make(chan os.Signal, 1)
-	signal.Notify(waitCtrlC, os.Interrupt)
-	<-waitCtrlC
+	// wait for messages
+	done := make(chan bool, 1)
+	go func() {
+		for {
+		}
+	}()
+	<-done
```

Comment:
@yaacov this works, but it's ugly.
Do you have something better?

Reply:
> but it's ugly.

first we merge, then we will think of something better :-)